### ISSUE TITLE
Need user interface for setting up Mailgun

### ISSUE THREAD

**Issue by bschipul** [tendenci/tendenci#429]
We need to be able to set up Mailgun entries in the site settings. Setup should not involved going into terminal.

**Comment by eschipul** [comment]
Agreed. It would be helpful if you mentioned the current method and variable names so others might be able to look at it as well. I think in T5 it is in .env and in t7 honestly don't recall. A bit more detail if possible?
